GatewayMicroService/GatewayMicroService/source/api/parser_api_router.py

```diff
@@ -0,0 +1,56 @@
+from fastapi import APIRouter, Request,HTTPException,UploadFile,File,Form
+import httpx
+import json
+
+from source.services.request_handler import  json_data
+
+router = APIRouter()
+
+SECOND_MICROSERVICE_URL = "http://localhost:8001"  # Assuming the second microservice is running on this URL
```

Comment:
add it into configuration class and get it from there. also make it configurable